src/overlays/actors/ovl_En_Holl/z_en_holl.c

```diff
@@ -324,20 +409,20 @@ void EnHoll_Update(Actor* thisx, PlayState* play) {
 void EnHoll_Draw(Actor* thisx, PlayState* play) {
     EnHoll* this = (EnHoll*)thisx;
     Gfx* gfxP;
-    u32 setupDlIdx;
+    u32 setupDlIndex;
 
     // Only draw the plane if not invisible
     if (this->planeAlpha != 0) {
         OPEN_DISPS(play->state.gfxCtx, "../z_en_holl.c", 805);
 
         if (this->planeAlpha == 255) {
             gfxP = POLY_OPA_DISP;
-            setupDlIdx = SETUPDL_37;
+            setupDlIndex = SETUPDL_37;
         } else {
             gfxP = POLY_XLU_DISP;
-            setupDlIdx = SETUPDL_0;
+            setupDlIndex = SETUPDL_0;
         }
-        gfxP = Gfx_SetupDL(gfxP, setupDlIdx);
+        gfxP = Gfx_SetupDL(gfxP, setupDlIndex);
```

Comment:
```suggestion
        gfxP = Gfx_SetupDL(gfxP, setupDLIndex);
```
So it's like `Gfx_SetupDL_XXX`?

Reply:
sure, I often miss Dl vs DL